components/nds-prev-next/src/PrevNext.js

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import PropTypes from "prop-types";
+import classnames from "classnames";
+import "./../scss/prev-next.scss";
+
+const Link = ({ text, destination, elementType: LinkTag = "a", intro }) => {
+	let linkProps = {
+		className: "prev-next__link"
+	};
+	if (LinkTag === "a") {
+		linkProps.href = destination;
+	} else {
+		linkProps.to = destination;
+	}
+	return (
+		<LinkTag {...linkProps}>
+			<span className="prev-next__link-intro">{intro}</span>
+			<span className="prev-next__link-text">{text}</span>
+		</LinkTag>
+	);
+};
+
+export const PrevNext = props => {
+	const { nextPageLink, previousPageLink, className, ...rest } = props;
+
+	const nextLinkProps = {
+		intro: nextPageLink?.intro || "Next page",
+		text: nextPageLink?.text || nextPageLink?.destination,
```

Comment:
Not sure we should be using the destination as the fallback for link text? Isn't it better to just have the intro without text?